Majasdarbs/03-majasdarbs/cv-css/css/style.css

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
+section {
+    padding: 50px;
+    min-height: 10vh;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    max-width: 900px;
+    margin: auto;
+  }
+  hr{
+    max-width: 900px;
+  }
+  address{
+    max-width: 900px;
+  }
+  body {
+    background-color: #252422;
+    color: rgb(189, 189, 189);
+    font-family: Consolas,monaco,monospace;
+    margin: auto;
+  }
+  h1 {
+    font-weight: 700;
+    font-size: 2rem;
+    font-style: normal;
+  }
+  #hero-title-name{
+    font-family: 'Bebas Neue', cursive;
+    font-size: 80px;
+    flex-wrap: nowrap;
+    text-align: left;
+    display: flex;
+    margin: 0;
+    color: #fff;
+  }
+  #hero-title-position{
+    font-family: 'Bebas Neue', cursive;
+    font-size: 300%;
+    flex-wrap: nowrap;
+    text-align: left;
+    display: flex;
+    margin: 0;
+    color: rgb(189, 189, 189);
+  }
+  #aboutme{
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+  }
+  #right-container{
+    width: 50%;
+    min-height: 250px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    justify-content: center;
+    font-family: 'Bebas Neue', cursive;
+    font-size: 50px;
+    text-decoration: underline;
+  }
+  
+  /* HOVERS */
+  #right-container a:hover,
+  header a:hover{
+    color: #a72f00;
+  }
+  #left-container svg:hover{
+    fill: #a72f00;
+  }
+  /* /HOVERS */
+  #left-container{
+    width: 50%;
+    min-height: 250px;
+    display: flex;
+    flex-direction: column;
+  }
+  #social-icons{
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: flex-start;
+    min-height: auto;
+    padding:  0;
+    width: 100%;
+  }
+  #social-icons a{
+    padding-right: 30px;
+  }
+  h2 {
+    font-size: 1.8rem;
+  }
+  a {
```

Comment:
try moving theme styles to the document top.
You might also consider creating theme.css
Where you specify tags/common styles for your app.